sdk/communication/communication-phone-numbers/.vscode/launch.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
Remove this file.

Reply:
Is this a new guidline? A lot of other libraries include a launch.json and it's useful for debugging in vscode.